libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-datepicker/cell-editor-datepicker.component.ts

```diff
@@ -29,16 +35,23 @@ export class SkyAgGridCellEditorDatepickerComponent
   implements ICellEditorAngularComp
 {
   public columnWidth: number;
-  public currentDate: Date;
```

Comment:
Removing public field is a breaking change.

Reply:
Same comment about the component not being public.